ui/src/main/java/com/xmartlabs/base/ui/mvp/MvpView.java

```diff
@@ -1,4 +1,4 @@
-package com.xmartlabs.template.ui.mvp;
```

Comment:
I think that after this pull be merged we should separate the MVP architecture in a new modue